app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: "Cafe") -> None:
+    nodle = True
```

Comment:
The variable name `nodle` is unclear. Consider renaming it to something more descriptive, such as `all_compliant`, to better convey its purpose.